app/models/story_parser.rb

```diff
@@ -5,7 +5,6 @@ class StoryParser
   require 'timeout'
   require 'nokogiri'
   require 'mechanize'
-  require 'open-uri'
```

Comment:
We are actually still invoking open-uri below, so I'm not sure you want to remove that.